network/noise/src/noise_fuzzing.rs

```diff
@@ -0,0 +1,293 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//
+// Noise Fuzzing
+// =============
+//
+// This fuzzes the wrappers we have around the Noise library `snow`.
+//
+
+use super::Handshake;
+use crate::{NoiseParams, NoiseSocket, NOISE_PARAMETER};
+use futures::{
+    executor::block_on,
+    future::join,
+    io::{AsyncRead, AsyncWrite, Result as futuresResult},
```

Comment:
note: `futures::io::Result` is just a type alias for `std::io::Result`, so instead of 

```rust
use futures::io::Result as futuresResult;
```

we've been doing

```rust
use std::io;
```

and then use `io::Result` instead of `futuresResult` everywhere